### DISCUSSION TITLE
How to open tensorboard in the workspace like VSCode?

### DISCUSSION THREAD

**Discussion by MrWiffer** [coder/code-server#7124] [Q&A]
How can code server open TensorBoard in the workspace and display it normally like VSCode? Can I only open it in the browser through the code server prompt now?
![image](https://github.com/user-attachments/assets/0d53bbe5-289b-4131-943b-513a8372eb9e)

**Comment by benz0li** [comment]
Try setting `TENSORBOARD_PROXY_URL`. Adapt to your deployment.

Cross references:

* https://github.com/jupyter/docker-stacks/pull/2126

**Comment by benz0li** [comment]
Nevermind. Same reason as https://github.com/coder/code-server/discussions/7129: No HTTPS (HTTP only works for `127.0.0.1` / `localhost`).

Cross references:

* https://github.com/coder/code-server/issues/4395#issuecomment-1709901976
  * https://github.com/coder/code-server/issues/5475#issuecomment-1229529712
